tests/openvino/test_modeling.py

```diff
@@ -2969,3 +2970,75 @@ def test_huggingface_pipeline_streaming(self, model_arch):
 
         del hf_pipe
         gc.collect()
+
+
+class OVModelForTextToSpeechSeq2SeqIntegrationTest(unittest.TestCase):
+    SUPPORTED_ARCHITECTURES = ("speecht5",)
+
+    def _generate_text(self):
+        return "This text is converted to speech using OpenVINO backend"
+
+    def _generate_speaker_embedding(self):
+        np.random.seed(42)
+        speaker_embedding = np.random.randn(1, 512).astype(np.float32)
+        return torch.tensor(speaker_embedding)
+
+    def _get_processor(self, model_id, model_arch):
+        if model_arch == "speecht5":
+            from transformers import SpeechT5Processor
+
+            processor = SpeechT5Processor.from_pretrained(model_id)
+            return processor
+        else:
+            raise Exception("{} unknown processor for text-to-speech".format(model_arch))
+
+    def _get_model(self, model_id, model_arch):
+        if model_arch == "speecht5":
+            from transformers import SpeechT5ForTextToSpeech
+
+            model = SpeechT5ForTextToSpeech.from_pretrained(model_id)
+            return model
+        else:
+            raise Exception("{} unknown model for text-to-speech".format(model_arch))
```

Comment:
not sure if these check for unsupported models are needed as test design pattern uses supported models list for parametrization, and it guarantees that nothing unsupported by tests will be passed

Reply:
Don't think that there is smth bad to have this check. Imagine, someone adds new arch into that list but forgot to update this test to load the model. Then this check will mention it.